src/azure/msal/msalAzureAuth.ts

```diff
@@ -362,7 +380,7 @@ export abstract class MsalAzureAuth {
     }
 
     private async makeGetRequest<T>(
-        uri: string,
+        requestUrl: string,
```

Comment:
This is a port of the same code that worked in ADS, right?

Reply:
That's correct.